common/djangoapps/contentserver/test/test_contentserver.py

```diff
@@ -56,11 +62,13 @@ def setUpClass(cls):
         # A locked asset
         cls.locked_asset = cls.course_key.make_asset_key('asset', 'sample_static.txt')
         cls.url_locked = unicode(cls.locked_asset)
+        cls.url_locked_versioned = get_versioned_asset_url(unicode(cls.locked_asset))
```

Comment:
To answer the first question, though: I simply copypasta'd, so yeah, I can avoid generating it.  I'll clean that up.